dandi/publish/tests/test_asset.py

```diff
@@ -73,3 +73,44 @@ def test_asset_rest_retrieve(api_client, asset):
         'updated': TIMESTAMP_RE,
         'metadata': asset.metadata,
     }
+
+
+@pytest.mark.django_db
+def test_asset_path_filter(api_client, asset):
```

Comment:
Do you think it's reasonable to include any test cases where the filter doesn't match? How about ones where the case-insensitivity is tested?

Reply:
I'm hoping these could maybe be parameterized, perhaps with a bool parameter for whether each sub-case passes.